governance/votes/JanSky Executive Exercise June 12, 2023.md

```diff
@@ -0,0 +1,168 @@
+
+| Vault    | Current SF | Proposed SF |
+|----------|------------|-------------|
+| ETH-A    | 3.74%      | **3.44%**   |
+| ETH-B    | 4.24%      | **3.94%**   |
+| ETH-C    | 3.49%      | **3.19%**   |
+| WSTETH-A | 3.74%      | **3.44%**   |
+| WSTETH-B | 3.49%      | **3.19%**   |
+| RETH-A   | 3.74%      | **3.44%**   |
+| WBTC-A   | 5.80%      | **5.69%**   |
+| WBTC-B   | 6.30%      | **6.19%**   |
+| WBTC-C   | 5.55%      | **5.44%**   |
+
+### Spark Protocol Proxy Spell
+
+As per this forum [post](https://forum.makerdao.com/t/freeze-the-sdai-market-on-spark/21322), a proxy spell will be executed to freeze the sDAI market on Spark Protocol, if this executive proposal passes.
+
+"The purpose of this update is to restrict sDAI as collateral to only short ETH and staked ETH. Freezing a market prevents new deposits, but users are always free to repay/withdraw."
+
+The spell code for the proxy spell can be found at [0x843A0539Ca7466Abcb769f1c1d30C8423e13A297](https://etherscan.io/address/0x843A0539Ca7466Abcb769f1c1d30C8423e13A297#code).
+
+## Minor Proposal Details
+
+### Budget Management Actions
+
+As per this forum [post](https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306) the following budget actions will take place, if this executive proposal passes.
+
+#### Chronicle Labs
+
+- Chronicle Labs Auditor Wallet will be streamed **3,721,800 DAI** from 2023-07-01 00:00:00 to 2024-06-30 23:59:59 at [0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f).
+- Chronicle Labs Auditor Wallet will be streamed **2,216.4 MKR** from 2023-07-01 00:00:00 to 2024-06-30 23:59:59 at [0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f).
+
+#### Jetstream 
+
+- Jetstream Auditor Wallet will be streamed **2,964,006 DAI** beginning on 2023-07-01 and ending on 2024-12-31 to [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962).
+- Jetstream Auditor Wallet will be streamed **1,619.93 MKR** begining 2023-06-26 and ending on 2024-12-31 to [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962).
+- Jetstream will receive 494,001 DAI to [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962).
+
+#### Development and UX Core Unit (DUX-001)
+- DUX-001 will receive 56.48 MKR at [0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad](https://etherscan.io/address/0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad).
+- End (yank) DAI stream 14 when this executive passes. 
+
+#### Deco Core Unit (DECO-001)
+
+- DECO-001 will receive 125 MKR at [0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7).
+
+
+### June Awards for Aligned Delegates
```

Comment:
Probably wouldn't term it 'awards', it doesn't have quite the correct connotations.